project-reports-lib.ts

```diff
@@ -8,7 +8,43 @@ export * from './project-reports-schemes';
 // filter cards by label
 //
 export function filterByLabel(cards: IssueCard[], name: string): IssueCard[] {
-    return cards.filter((card) => card.labels.findIndex(label => label.name === name) >= 0);
+    return cards.filter((card) => card.labels.findIndex(label => label.name.toLowerCase() === name.toLowerCase()) >= 0);
+}
+
+//
+// Calculate cycle time for a card
+// The time, in days, a unit of work spends between the first day it is actively being worked on until the day it is closed.
+// In this case, since a project card has events, we look for the event that moved or added a card to the "Accepted" column
+// and subtract it from the time that the card moved to the `Done` column.
+//
+export function calculateCycleTime(card: IssueCard): number {
```

Comment:
Does this need to be in the lib or should it be in the cycle time report as a private function?

Reply:
I can move it to the cycle time report generator.